app/jobs/system_sms_job.rb

```diff
@@ -0,0 +1,33 @@
+class SystemSMSJob < ApplicationJob
+  queue_as :default
+
+  def perform(message, recipients, current_user)
+    errors = []
+    recipients.each do |user_id|
+      user = User.find(user_id)
+      phone_number = if user.mobile_phone.present?
```

Comment:
What did you find on other usages about phone number? We may want to only use mobile_phone.

Reply:
i misunderstood, mobile_phone should be all thats needed